pkg/portal/graphql/collaborator_mutation.go

```diff
@@ -110,6 +111,23 @@ var createCollaboratorInvitationInput = graphql.NewInputObject(graphql.InputObje
 	},
 })
 
+var createCollaboratorInvitationInputSchemaName = "CreateCollaboratorInvitationInputSchema"
+
+var createCollaboratorInvitationInputSchema = validation.NewMultipartSchema("").
```

Comment:
Actually I'm thinking to replace most of usage of multipart schema with simple schema, since they just have single schema. What do you think?

Reply:
We can keep the existing multipart schema API. The single schema API should be a shorthand over the current API.

Reply:
When I was developing the validation schema abstraction, I was thinking of using multipart schema only for the user-facing schemas (i.e. app/secret configs), while the ad-hoc validation schemas would use simple schema for simplicity.

Reply:
Can you open an issue to track this?

Reply:
ref #513